opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -104,6 +133,7 @@ message Metric {
   repeated DoubleDataPoint double_data_points = 3;
   repeated HistogramDataPoint histogram_data_points = 4;
   repeated SummaryDataPoint summary_data_points = 5;
+  repeated RawValue raw_data_points = 6;
```

Comment:
I think this should be removed for the moment, this is not exemplar, is supporting "RawMeasurements" which is out of scope for this PR.

Reply:
removed